_data/education.yml

```diff
@@ -51,6 +51,13 @@ websites:
       twitter: duolingo
       tfa: No
 
+    - name: EC-Council
+      url: https://www.eccouncil.org
+      img: eccouncil.png
+      tfa: Yes
+      sms: Yes
+      doc: https://aspen.eccouncil.org/aspen/Help/2FactorAuthentication-User_Manual.pdf
```

Comment:
The documentation also specifies sending the verification code via email. See page 2 and 3. 

> 1. ...
> 2. ...
> 3. Select one of the options presented on the screen. Select My mobile number to receive
the verification code on your mobile number, My registered email to receive the ... verification code on your registered email, or No, I want to use a backup code if you have
a backup verification code, and click Use this method. 

Could you please add the following? 

```ruby
      email: Yes
```